apps/authentication-service/cypress/e2e/clients.cy.ts

```diff
@@ -0,0 +1,123 @@
+import { Client } from "auth0";
+
+const USER_ID: string = "auth0|667c3d0c90b963e3671f411e";
+
+describe("Client API", () => {
+  const PORT: number = (Cypress.env("PORT") as number) || 8089;
+  let accessToken: string;
+  let m2mToken: string;
+  let newClientId: string;
+
+  const getTokens = () => {
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-m2m")
+      .then((token: string) => {
+        if (token) {
+          m2mToken = token;
+        }
+      });
+    cy.window()
+      .its("localStorage")
+      .invoke("getItem", "jwt-client")
+      .then((token: string) => {
+        if (token) {
+          accessToken = token;
+        }
+      });
+  };
+
+  before(() => {
+    getTokens();
+    if (!accessToken) {
+      cy.loginM2MAuth0();
+      cy.wait(1500);
+    }
+    if (!m2mToken) {
+      cy.loginClientAuth0();
+      cy.wait(1500);
+    }
+    getTokens();
+  });
+
+  // "creates a client, fails if another client request is made, gets the newly created client, and deletes the client"
+  it("clears the client_id from user metadata, tries creating a client without access token, creates a client, fails if another client request is made, gets the newly created client, and deletes the client", () => {
+    cy.request({
+      method: "PATCH",
+      url: `https://${Cypress.env("AUTH0_DOMAIN")}/api/v2/users/${USER_ID}`,
+      body: {
+        user_metadata: {
+          client_id: "",
+        },
+      },
+      headers: {
+        Authorization: `Bearer ${m2mToken}`,
+      },
+    });
+
+    cy.request({
```

Comment:
I'd put this in another it statement. It doesn't require any setup or teardown, so it's easy to separate.